common/src/main/java/com/linecorp/line/auth/fido/fido2/common/mdsv3/MetadataBLOBPayloadEntry.java

```diff
@@ -0,0 +1,20 @@
+package com.linecorp.line.auth.fido.fido2.common.mdsv3;
+
+import com.linecorp.line.auth.fido.fido2.common.mdsv3.metadata.MetadataStatement;
+import lombok.Data;
+
+import java.util.List;
+@Data
+public class MetadataBLOBPayloadEntry {
+    private String aaid;
+    private String aaguid;
+    private List<String> attestationCertificateKeyIdentifiers;
+    private MetadataStatement metadataStatement;
+    private List<String> biometricStatusReports;
```

Comment:
This is supposed to be a dictionary with multiple entries:
https://fidoalliance.org/specs/mds/fido-metadata-service-v3.0-ps-20210518.html#dictdef-biometricstatusreport

Is there any reason why it is parsed as a String here?

Reply:
Fixed in 17a944d.